docs/guides/primitives-rest-api.mdx

```diff
@@ -479,6 +479,141 @@ else:
   </TabItem>
 </Tabs>
 
+<span id="start-sampler-parms"></span>
+## Sampler primitive with REST API and parameterized circuits
+
+### 1. Initialize the account
+
+Because Qiskit Runtime Sampler is a managed service, you first need to initialize your account. You can then select the device you want to use to run your calculations on.
+
+Find details on how to initialize your account, view available backends, and invalidate tokens in this [topic](./setup-channel#set-up-to-use-ibm-quantum-platform-with-rest-api).
+
+### 2. Define parameters
+
+```python
+import requests
+import qiskit_ibm_runtime
+# Imports from Qiskit Runtime
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+
+from qiskit.qasm3 import dumps
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit import transpile
+
+service = QiskitRuntimeService(channel='ibm_quantum')
+backend = service.backend("your selected backend")
+
+pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+
+theta = Parameter('theta')
+phi = Parameter('phi')
+parameter_values = {'theta': 1.57, 'phi': 3.14}   # In case we want to pass a dictionary
+```
+
+
+### 3. Create a quantum circuit and add parameterized gates
+
+```python
+qc = QuantumCircuit(2)
+
+# Add parameterized gates
+qc.rx(theta, 0)
+qc.ry(phi, 1)
+qc.cx(0, 1)
+qc.measure_all()
+
+# Draw the original circuit
+qc.draw('mpl')
+
+# Get an ISA circuit
+isa_circuit= pm.run(qc)
+```
+
+### 4. Generate QASM 3 code
+
+```python
+qasm_str = dumps(isa_circuit)
+print("Generated QASM 3 code:")
+print(qasm_str)
+```
+
+### 5. Run the quantum circuit using Sampler V2 API
+
+
+<Admonition type="note">
+    The jobs below use [Qiskit Runtime V2 primitives](/migration-guides/v2-primitives). Both `SamplerV2` and `EstimatorV2` take one or more primitive unified blocs (PUBs) as the input. Each PUB is a tuple that contains one circuit and the data broadcasted to that circuit, which can be multiple observables and parameters. Each PUB returns a result.
```

Comment:
Should this be `/guides/primitives`? Also link to the API docs for Sampler and Estimator? Link to the PUB file?